UniShed/univer_structure/models.py

```diff
@@ -111,9 +111,9 @@ class DegreesOfStudent(models.TextChoices):
         MASTERS_DEGREE = 'Магистратура', _('Магистратура')
         PHD_DEGREE = 'Аспирантура', _('Аспирантура')
 
-    name_degree = models.CharField(max_length=10,
+    name_degree = models.CharField(max_length=30,
                                      choices=DegreesOfStudent.choices,
-                                     default=DegreesOfStudent.FIRST_COURSE)
+                                     default=DegreesOfStudent.BACHELOR_DEGREE)
 
 
 class CourseNumber(models.Model):
```

Comment:
what is this model for?

Reply:
For storing the numbers of courses that students' need to study for the degree's program . If we need to specify that some groups are in the first course of Bachelor degree with specialty "IT" it would be better to attach that course number to discipline that they should study